x-pack/test/security_api_integration/tests/user_profiles/get_current.ts

```diff
@@ -34,7 +34,7 @@ export default function ({ getService }: FtrProviderContext) {
         password: 'changeme',
         roles: [`viewer`],
         full_name: 'User With Profile',
-        email: 'user_with_profile@elastic.co',
+        email: 'user_with_profile@get_current_test',
```

Comment:
__note:__ just to make running these tests locally easier since we don't have proper API to remove user profiles yet (we can just disable them currently).